### PR TITLE
Added tests and meta.yml files for Salmon

### LINKED ISSUE #102: Add tests for salmon

### PR DESCRIPTION
**Pull request by KevinMenden**
Added tests and meta.yml files for Salmon.

Closes #102 

I had to add more data unfortunately, chose some covid data because of the smaller size of viral data.
Might update the STAR (and maybe also bowtie etc.) tests some day to also use this data and safe some space. But it should be okay for now.

### CODE REVIEW

File: software/salmon/index/meta.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: salmon_index
+description: Create index for salmon
+keywords:
+  - index
+  - fasta
+  - genome
+  - reference
+tools:
+  - star:
```
Comment (drpatelh) [inline software/salmon/index/meta.yml:9]:
```suggestion
  - salmon:
```

File: software/salmon/quant/meta.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+name: salmon_quant
+description: gene/transcript quantification with Salmon
+keywords:
+  - index
+  - fasta
+  - genome
+  - reference
+tools:
+  - star:
```
Comment (drpatelh) [inline software/salmon/quant/meta.yml:9]:
```suggestion
  - salmon:
```

**Review by drpatelh — CHANGES_REQUESTED** [review]
Looks great! Thanks! Can we rename the tests folder to `sarscov2/` instead of `covid`? The disease is COVID-19 and not the name of the virus. Great idea for using the viral data for testing though 😎

File: software/salmon/index/meta.yml (hunk 1 above)
Comment (KevinMenden) [reply software/salmon/index/meta.yml:9]: oops

**Comment by KevinMenden** [comment]
Thanks - yes you're right :) makes much more sense! will change the name

**Review by drpatelh — APPROVED** [review]

**Comment by drpatelh** [comment]
Thanks @KevinMenden !

**Pull request merged**